editor/plugins/org.fusesource.ide.projecttemplates/OSGI-INF/l10n/bundle.properties

```diff
@@ -1,36 +1,37 @@
+Bundle-Vendor = JBoss by Red Hat
+Bundle-Name = Fuse Project Templates Plugin
+configurator.name = JBoss Fuse m2e Project Configurator
+
+jboss.fuse.category.label = JBoss Fuse
+project.wizard.name = Fuse Integration Project
+project.wizard.description = Create a new Fuse Integration Project using Maven as build system
+
+template.category.default = Other
+template.category.jbossfuse = JBoss Fuse
+template.category.jbossfuse.simple = Beginner
+template.category.jbossfuse.advanced = Advanced
+template.category.jbossfuse.expert = Expert
 template.category.eap = Fuse on EAP
 template.category.eap.medium = Medium
-template.category.ose = Fuse on OpenShift
-
-template.simple.cbr.name = Content Based Router
-template.simple.cbr.description = This template provides a sample Camel route definition which reads files from a specific location and routes these files depending on their content to different output folders. 
-template.simple.amq.name = ActiveMQ
-template.simple.amq.description = This template provides a sample Camel route definition which reads files from a specific location and puts them into a message queue. Another route pulls the messages from the queue and does a content based routing.\n\nWARNING: to deploy on JBoss Fuse, a property file need to be configured. Please see ReadMe.txt at the root of the created project for more details.
+template.category.ose = Fuse on OpenShift
+
+template.simple.cbr.name = Content Based Router
+template.simple.cbr.description = This template provides a sample Camel route definition which reads files from a specific location and routes these files depending on their content to different output folders. 
+template.simple.amq.name = ActiveMQ
+template.simple.amq.description = This template provides a sample Camel route definition which reads files from a specific location and puts them into a message queue. Another route pulls the messages from the queue and does a content based routing.\n\nWARNING: to deploy on JBoss Fuse, a property file need to be configured. Please see ReadMe.txt at the root of the created project for more details.
 template.medium.eapspring.name = Spring on EAP
-template.medium.eapspring.description = This template provides a sample Camel route definition which calls into a bean service to say hello.
+template.medium.eapspring.description = This template provides a sample Camel route definition which calls into a bean service to say hello.
 template.medium.cxf.code.first.name = CXF code first
-template.medium.cxf.code.first.description = This template provides a sample Camel route definition which is started by a CXF Webservice call.
-extension-point.name = projectTemplates
\ No newline at end of file
+template.medium.cxf.code.first.description = This template provides a sample Camel route definition which is started by a CXF Webservice call.
+extension-point.name = projectTemplates
+integration.preferences.project.templates=Project Templates
```

Comment:
it is nto affecting only Project Templates, it is also used when the facet version is applied/modified.
Using aa name such as "Staging repositories" might be more precise